js/output/webpage/webpage-output.js

```diff
@@ -198,12 +198,13 @@ window.WebpageOutput = Backbone.View.extend({
             HTTP_LINK_FROM_HTTPS_PAGE: $._("The <%(openTag_name)s> tag's \"%(attribute_name_value)s\" attribute currently points to an insecure resource.", error),
             INVALID_URL: $._("The <%(openTag_name)s> tag's \"%(attribute_name_value)s\" attribute points to an invalid URL.  Did you include the protocol (http:// or https://)?", error),
             INVALID_ATTR_NAME: $._("The attribute name \"%(attribute_name_value)s\" is not permitted under HTML5 naming conventions.", error),
+            OBSOLETE_HTML_TAG: $._("%(openTag_name)s tags are obsolete and may not function in modern browsers", error),
```

Comment:
I'll add the period. I'm using "obsolete" because deprecated tags require continued browser support and obsolete tags do not (http://stackoverflow.com/a/11785062/4147306).